new-website/deepchem/styles/models.css

```diff
@@ -1,48 +0,0 @@
-@tailwind base;
```

Comment:
Why are we deleting this CSS file?

Reply:
It looks like we moved this over to globals.css is that right?

Reply:
Yes, we have shifted the models.css content to global.css. I am not sure exactly why, but the previous structure caused some of the styling of the homepage to be overwritten. Shifting the same solves the problem.

Reply:
I have been trying to find the exact reason for this issue, but found nothing concrete till now.

Reply:
Ok sounds good. Can you add a screenshot for the rendered site to this issue? I think we should be good to merge after

Reply:
The explore heading is not being styled correctly

![Explore-heading-not-properly-rendered](https://user-images.githubusercontent.com/74804069/225835089-f7bb71b2-a2e3-4caa-bc24-9a629d4d733b.jpeg)

The expected styling is

![correct-rendering](https://user-images.githubusercontent.com/74804069/225835477-79978793-3531-4956-8545-34d0b99bf4a3.png)